### PR TITLE
feature/improve-use-state-snippet

### PR DESCRIPTION
**Pull request by yuritoledo**
Improve the UseState snippet, to a better DX.
Was fixed some duplicated keys too.

PS: Some indentation was with tabs among the spaces, was fixed too.

### CODE REVIEW

File: snippets/snippets.json [hunk 1]
```diff
@@ -1009,24 +1189,36 @@
...
+      "}).isRequired,"
+    ],
     "description": "An object taking on a particular shape required"
   },
   "staticPropTpyes": {
     "prefix": "ptypes",
-    "body": ["static propTypes = {", "$0", "}", ""]
+    "body": [
+      "static propTypes = {",
+      "$0",
```
Comment (dsznajder) [inline snippets/snippets.json]: Missing 2 spaces :)

File: snippets/snippets.json [hunk 2]
```diff
@@ -1204,7 +1412,9 @@
   },
   "useState": {
     "prefix": "useState",
-    "body": ["const [${1:state}, ${2:setState}] = useState(${3:initialState})"]
+    "body": [
+      "const [${1:state}, set${1/(.*)/${1:/capitalize}/}] = useState($2:initialState)"
```
Comment (dsznajder) [inline snippets/snippets.json]: Nice :)

Comment (yuritoledo) [reply snippets/snippets.json]: Thanks!

**Comment by yuritoledo** [comment]
I just use the VSCode inner formatter :)

**Pull request merged**